frontend/components/panel/js/panel.component.ts

```diff
@@ -129,7 +129,7 @@ class PanelController implements ng.IController {
 			// If we have clicked on a canvas, we are probably moving the model around
 			if (event.target.tagName === "CANVAS") {
 				this.activate = false;
-				this.$scope.$apply();
+				//this.$scope.$apply();
```

Comment:
remove unless there is a reason why it needs to stay (in which case we need a comment here)